deps/tools/uninstallers.mk

```
@@ -28,4 +28,5 @@ $(addprefix version-check-,$(DEP_LIBS_STAGED)) : version-check-% : install-%
 			[ "$(UNINSTALL_$*)" != "`cat $(build_prefix)/manifest/$*`" ]) ; then \
 		echo "WARNING: using mismatched version for $$(cat $(build_prefix)/manifest/$*):" ; \
 		echo "  want $(UNINSTALL_$*)" ; \
+		echo "  Try deleting the usr directory." ; \
```

```suggestion
		echo "  Try deleting the $(JULIAHOME)/usr directory." ; \
```

Let's not leave any ambiguity as to which `usr` directory we're advocating the deletion of.  ;)

---

Tries incorrectly to test on my system...
```
$ echo $(JULIAHOME)/usr 
/usr
```

Yeah, let's leave no doubt 😂

We could also suggest some of the less nuclear options (e.g. `make -C deps uninstall`)?

---

"Try deleting the usr subdirectory in the build directory"?

---

@staticfloat @vtjnash I have made the path of the usr directory more explicit. What do you think?

I also added @vtjnash's suggestion for a less drastic option

---

I’m not sure Jameson was serious about not using the actual $(JULIAHOME) directory, I think he was making a joke about copy-pasting the value into his terminal from the makefile source. If you use the actual variable value, it will print out the actual folder that should be deleted, which seems like the best option overall.

---

Ah hahaha whoops my bad. Okay let me fix that. Should I leave in his suggestion about `make -C deps uninstall`?

---

I've fixed the message to use the actual value of the `JULIAHOME` variable.